compiler/pipes/generate-virtual-methods.cpp

```diff
@@ -423,7 +423,7 @@ void generate_body_of_virtual_method(FunctionPtr virtual_function) {
     }
   }
   if (!cases.empty()) {
-    auto case_default_warn = generate_critical_error_call(fmt_format("call method({}) on null object", virtual_function->as_human_readable()));
+    auto case_default_warn = generate_critical_error_call(fmt_format("call method({}) on null object", virtual_function->as_human_readable(false)));
```

Comment:
Why?

Reply:
For codegeneration stability. "true" outputs instantiation location, which is ok for human-readable (for compilation errors, etc.), but is unstable and leads to occasional changes C++ code.